apps/voting/contracts/Voting.sol

```diff
@@ -226,7 +248,7 @@ contract Voting is IForwarder, AragonApp {
     }
 
     /**
-    * @notice Tells whether `_sender` can participate in the vote #`_voteId` or not
+    * @notice Tells whether `_sender` can vote in the vote #`_voteId` by voting 'yes' or 'no'
     * @dev Initialization check is implicitly provided by `voteExists()` as new votes can only be
     *      created via `newVote(),` which requires initialization
     * @return True if the given voter can participate a certain vote, false otherwise
```

Comment:
@return True if the given voter can participate in the main phase of a certain vote, false otherwise